api/onnx_web/chain/base.py

```diff
@@ -105,27 +105,19 @@ def __call__(
                 len(sources),
             )
         else:
+            sources = [None]
             logger.info("running pipeline without source images")
 
         stage_sources = sources
         for stage_pipe, stage_params, stage_kwargs in self.stages:
             name = stage_params.name or stage_pipe.__class__.__name__
             kwargs = stage_kwargs or {}
             kwargs = {**pipeline_kwargs, **kwargs}
-
-            if len(stage_sources) > 0:
```

Comment:
I see that the change on ln#108 will break this `if` check, but I think we should include some information about the number of sources in the log message. It can be helpful when you have a txt2img with batch size > 1 followed by highres, for example.

Reply:
Fixed